### DISCUSSION TITLE
Nouveau Nspanel Pro 120 WOUAOUH ! - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#58] [Comments]
# Nouveau Nspanel Pro 120 WOUAOUH ! - Haade.fr

Un test attendu avec impatience celui du tout nouveau sonoff Nspanel pro 120 PW, un format qui fait l’unanimité mais il n’y a pas que la taille qui compte. Je vais tu t’en doute le comparer au NsPanel Pro 86

https://haade.fr/fr/blog/nouveau-sonoff-nspanel-pro-120-pw-wouaouh-que-du-bonheur

**Comment by mikaYX** [comment]
J'en ai acheter 4 pour en installer un peu partout chez moi. Je trouve que le potentiel avec home assistant est vraiment top, c'est al même procédure que sur le modèle 86 en grand format d'image :)

**Reply by haade-administrator** [reply]
c'est ça
